apps/rundown/src/launcher.ts

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import moduleApi = require('module');
```

Comment:
`import * as moduleApi from 'module';`?

Reply:
@iclanton That doesn't work, I think because `'module'` is a special undocumented Node.js API.